File: 0001 Code Chef/0002 Mango Lassi/implementation/0009 dart/mangolassi.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  int? temperature = int.tryParse(stringInput!);

  if(temperature != null){
    if(temperature > 35){
      print("YES");
    }else{
      print("NO");
    }
  }
}

File: 0001 Code Chef/0002 Mango Lassi/implementation/0010 nodejs/mango_lassi.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    temperature = parseInt(input);
    if(temperature > 35){
        console.log("YES");
    }else{
        console.log("NO");
    }
    rl.close();
})